### ISSUE TITLE
Inspector doesn't update fields when switching between volumes

### ISSUE THREAD

**Issue by kephale** [scenerygraphics/sciview#251]
I loaded 3 volumes, changed the scale of 1, when i then select a different volume, the scale value remains the same (e.g., the changed value).

**Comment by xulman** [comment]
on master at commit d4f219d2b1, when I call `addVolume()` and `setName(..)` the new name is not shown in the inspector, unless I make something visible off and on again

**Comment by xulman** [comment]
and relatedly, when I `addChild()` to the `Volume` node, the added nodes are not visible in the inspector until I add something (directly under the root of the scene)

**Comment by kephale** [comment]
Unsure if these are related, but https://github.com/scenerygraphics/sciview/issues/422 is another case of the same data being referenced from 2 separate `Volume` instances

**Comment by moreApi** [comment]
I cant reproduce the issue of the first post. 
What I tried:
- click twice on demo->basic->volume..
- change position and size of one
- switch between them using the tree view or double clicking in the scene

I always see the correct values in the property editor.
The only unexpected behavior I saw was that a new transfer function was applied to the changed volume. This is an issue but not this one.^^
